plugins/woocommerce-blocks/assets/js/blocks/customer-account/style.scss

```diff
@@ -16,35 +16,23 @@
 }
 
 .wp-block-woocommerce-customer-account {
-	display: flex;
-	align-items: center;
-	white-space: nowrap;
-	overflow: hidden;
-
 	a {
-		flex: 1;
-		overflow: hidden;
 		text-decoration: none !important;
 		align-items: center;
 		display: flex;
 		color: currentColor !important;
+		gap: em($gap-smaller);
+		white-space: nowrap;
+		line-height: 1;
+		padding: em($gap-smaller);
 
 		&:hover {
 			text-decoration: underline !important;
 		}
 
-		.icon {
-			height: em(16px);
-			width: em(16px);
-		}
-
 		.wc-block-customer-account__account-icon {
-			box-sizing: content-box !important;
```

Comment:
I think this line being removed is causing some issues I'm seeing on a site currently.

It may vary based on the theme, but if a theme is running a normalizer that does something like

```css
*, *::before, *::after {
    box-sizing: border-box;
    word-wrap: break-word;
}
```

that would mean that any padding added to the element under WooCommerce prior to this change would have not included the padding in the element's width (as it used to be content-box) but changing this to inherit border-box would squish the svg down and mess with its rendering when WooCommerce updated to include this change (without any action on the part of the user) leading to a regression.